docs/.vuepress/rules.js

```diff
@@ -74,6 +74,7 @@ module.exports = [
 'optional-if-present',
 'optional-if-present-or-else',
 'optional-map',
+'use-class-secure-random',
```

Comment:
This is not the correct place alphabetically. Please make sure to sort this list according to the title and not the file name (unfortunately, they sometimes differ for older rules).